content/docs/concurrent-mode-adoption.md

```diff
@@ -15,98 +15,97 @@ next: concurrent-mode-reference.html
+**Bu tüm `<App />` ağacı için eşzamanlı modu etkinleştirir:**
 
 ```js
 import ReactDOM from 'react-dom';
 
-// If you previously had:
+// Eğer eskiden
 //
 // ReactDOM.render(<App />, document.getElementById('root'));
 //
-// You can opt into Concurrent Mode by writing:
+// kullanıyorduysanız, eşzamanlı modu şunu yazarak etkinleştirebilirsiniz:
 
 ReactDOM.createRoot(
   document.getElementById('root')
 ).render(<App />);
 ```
 
->Note:
+>Not:
 >
->Concurrent Mode APIs such as `createRoot` only exist in the experimental builds of React.
+>`createRoot` gibi eşzamanlı mod APIları React'in sadece deneysel versiyonlarında bulunmaktadır.
 
-In Concurrent Mode, the lifecycle methods [previously marked](/blog/2018/03/27/update-on-async-rendering.html) as "unsafe" actually *are* unsafe, and lead to bugs even more than in today's React. We don't recommend trying Concurrent Mode until your app is [Strict Mode](/docs/strict-mode.html)-compatible.
+Eşzamanlı modda [daha önce](/blog/2018/03/27/update-on-async-rendering.html) "güvensiz" olarak işaretlenmiş yaşam döngüsü metotları bu sefer gerçekten *güvensizdir* ve bugünkü React'ten dahi daha çok hataya sebep olurlar. Uygulamanızın [Strict Mode](/docs/strict-mode.html) desteği olana dek eşzamanlı modu kullanmanızı önermiyoruz.
 
-## What to Expect {#what-to-expect}
+## Beklenmesi Gerekenler {#what-to-expect}
 
-If you have a large existing app, or if your app depends on a lot of third-party packages, please don't expect that you can use the Concurrent Mode immediately. **For example, at Facebook we are using Concurrent Mode for the new website, but we're not planning to enable it on the old website.** This is because our old website still uses unsafe lifecycle methods in the product code, incompatible third-party libraries, and patterns that don't work well with the Concurrent Mode.
+Eğer büyük bir uygulamanız varsa veya uygulamanızın çok fazla üçüncü parti paketlere bağımlılığı varsa, lütfen eşzamanlı modu anında kullanabileceğiziniz düşünmeyin. **Örneğin biz Facebook'ta eşzamanlı modu yeni sitede kullanıyoruz ama eski sitede kullanmayı planlamıyoruz.** Bunun nedeni, eski sitemizin hala güvensiz yaşam döngüsü metotlarını, uyumsuz üçüncü parti kütüphanelerini ve eşzamanlı modla çok iyi çalışmayan desenleri canlı kodda kullanıyor olması.
 
-In our experience, code that uses idiomatic React patterns and doesn't rely on external state management solutions is the easiest to get running in the Concurrent Mode. We will describe common problems we've seen and the solutions to them separately in the coming weeks.
+Bizim tecrübemiz, deyimsel React desenlerini kullanan ve harici state yönetimi çözümlerine bel bağlamayan kodun eşzamanlı modu çalıştırmada en kolay olduğu yönünde. Gördüğümüz ortak sorunları ve onların çözümlerini önümüzdeki haftalarda ayrıca anlatacağız.
 
-### Migration Step: Blocking Mode {#migration-step-blocking-mode}
+### Migrasyon Adımı: Engelleme Modu {#migration-step-blocking-mode}
```

Comment:
Buna çok emin olamadım. Diğer çevrilen sayfalarda migration için genelde  `geçiş` kelimesini kullanmışız. 

Geçiş Politikası , Geçiş için Yol Haritası vs.

Siz ne düşünüyorsunuz ?

Reply:
Geçiş uygun bence de, sorun değil. Önce göç diye çevirmiştim, ama bu daha az Türkçe olmasına karşın daha iyi geldi kulağıma.

Geçiş daha da iyiymiş, değiştiriyorum. Teşekkürler.